modzy/models.py

```diff
@@ -29,6 +31,115 @@ def __init__(self, api_client):
         self._api_client = api_client
         self.logger = logging.getLogger(__name__)
 
+    def get_model_details(self, model, version):
```

Comment:
this method brings the processing state of the model(s), so the name should be something like `get_model_processing_state` or something like that

Reply:
So this method actually provides the entire model details for a specific model and version if it is active. One part of the model details is the processing engines, but that is not the only output. This method is called as part of update_processing_engines() and that information is extracted to determine if the model is actively running or not